app/components/forms/wizard/basic-details-step.js

```diff
@@ -133,7 +133,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
 
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
-    $.fn.form.settings.rules.checkMaxMinPrice = () => {
+    $.fn.form.settings.rules.checkMaxMinPrice = function(min_price,max_price){
```

Comment:
Codacy found an issue: ['min_price' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5885738)